plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -454,6 +454,13 @@ static void get_nb_frames(void *data, calldata_t *cd)
 	calldata_set_int(cd, "pix_format", pix_format);
 }
 
+static void get_playing(void *data, calldata_t *cd)
+{
+	struct ffmpeg_source *s = data;
+	bool playing = s->media.playing;
```

Comment:
As media.playing flag not changed at the same time as other media status flags. For example in mp_media_stop() or in the mp_media_thread() it resets to false at the very end of function. This looks like possibility or potential possibility for race condition. As making more parts of code mutexed is not desirable so to make sure that playing will not be returned in situation when media already started stopping return value can be calculated like this. 

playing = s->media.active && !s->media.stopping && s->media.playing.